src/dbnode/client/connection_pool_test.go

```diff
@@ -47,9 +47,25 @@ var (
 
 type noopPooledChannel struct {
 	address string
+	closed  int32
```

Comment:
Mini naming nit: `closingCount`? As for `closed` I would expect a boolean type.

Reply:
yeah, it can be 1 or 0 only. There are no `atomic.LoadBool` in go :/

Reply:
We use https://github.com/uber-go/atomic/blob/master/bool.go